src/main/java/com/myodoctor/finalproject/services/Implementations/StaffServices.java

```diff
@@ -0,0 +1,42 @@
+package com.myodoctor.finalproject.services.Implementations;
+
+
+import com.myodoctor.finalproject.models.DeliveryPersonnel;
+import com.myodoctor.finalproject.models.Patient;
+import com.myodoctor.finalproject.models.Person;
+import com.myodoctor.finalproject.models.RegisterModel.DeliveryPersonnelRegistrationModel;
+import com.myodoctor.finalproject.models.RegisterModel.PatientRegistrationModel;
+import com.myodoctor.finalproject.models.RegisterModel.PersonRegisterModel;
+import com.myodoctor.finalproject.models.Staff;
+import com.myodoctor.finalproject.repositories.IDeliveryPersonnelRepositories;
+import com.myodoctor.finalproject.repositories.IStaffRepositories;
+import com.myodoctor.finalproject.services.Interfaces.IPersonServices;
+import com.myodoctor.finalproject.services.Interfaces.IStaffServices;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class StaffServices implements IStaffServices {
+
+    StaffServices staffServices;
+    IDeliveryPersonnelRepositories deliveryPersonnelRepositories;
+    DeliveryPersonnelRegistrationModel deliveryPersonnelModel;
+    Staff staff;
+    IPersonServices personServices;
+    PersonRegisterModel personModel;
```

Comment:
You don't have to inject register model that will be mapped with the form input field. You will pass it to the action method.